### PR TITLE
Update Languages/es.xml

### PR DESCRIPTION
**Pull request by Jhovany200**
Organized, and minor corrections.
Please, accept these corrections are the last.

### CODE REVIEW

File: Languages/es.xml [hunk 1]
```diff
@@ -36,48 +36,48 @@ Gracias a: Oleada, Jhovany200, nomemolesteis@gmail.com por la traducción.
...
-        <String Target="添加路径" Link="Añadir ruta"/>
-      <String Target="卸载映像&#9;Delete" Link="Desmotar imagen&#9;Supr"/>
-      <String Target="保存映像&#9;Ctrl+S" Link="Guardar imagen&#9;Ctrl+S"/>
-      <String Target="另存为映像&#9;Ctrl+Shift+S" Link="Guardar imagen como&#9;Ctrl+Mayús+S"/>
-      <String Target="打开映像文件&#9;Ctrl+O" Link="Abrir archivo de imagen&#9;Ctrl+O"/>
-      <String Target="重新载入" Link="Recargar"/>
-      <String Target="格式转换" Link="Convertir formato"/>
-        <String Target="ESD解密" Link="Desencriptar ESD"/>
-      <String Target="取消任务&#9;Ctrl+Break" Link="Cancelar tarea&#9;Ctrl+Break"/>
+    <String Target="添加映像" Link="Añadir imagen"/>
```
Comment (mingkuang-Chuyu) [inline Languages/es.xml]: Don't remove ' '...

File: Languages/es.xml [hunk 2]
```diff
@@ -116,17 +116,17 @@ Gracias a: Oleada, Jhovany200, nomemolesteis@gmail.com por la traducción.
     <String Target="正在加载规则" Link="Cargando reglas"/>
     <String Target="Dism++不允许修改文件名，这将导致插件无法正常加载。" Link="Dism++ no permite modificar el nombre del archivo, puede causar error al cargar el complemento."/>
     <String Target="无法启动Dism++x64.exe，请确保该Dism++组件的完整性。" Link="No se puede iniciar Dism++x64.exe, compruebe la integridad de Dism++."/>
-    <String Target="Dism++关键组件丢失，请从 www.chuyu.me 重新下载Dism++。" Link="Falta un archivo crítico en Dism++, vuelva a descargar Dism++ desde www.chuyu.me."/>
+    <String Target="Dism++关键组件丢失，请从 www.chuyu.me 重新下载Dism++。" Link="Falta un archivo crítico en Dism++, vuelva a descargar Dism++ desde (www.chuyu.me)."/>
```
Comment (mingkuang-Chuyu) [inline Languages/es.xml]: Falta un archivo crítico en Dism++, vuelva a descargar Dism++ desde \“www.chuyu.me\”.

?

File: Languages/es.xml (hunk 1 above)
Comment (Jhovany200) [reply Languages/es.xml]: do not remove it, it's just aligned.

File: Languages/es.xml (hunk 2 above)
Comment (Jhovany200) [reply Languages/es.xml]: ok

File: Languages/es.xml (hunk 1 above)
Comment (mingkuang-Chuyu) [reply Languages/es.xml]: I need a clear structure, not a single-minded alignment. Thanks

File: Languages/es.xml (hunk 2 above)
Comment (mingkuang-Chuyu) [reply Languages/es.xml]: I just have some questions, after all, you are the experts.

**Pull request closed without merging**